### PR TITLE
Use pass in logger for ml modules

### PR DESCRIPTION
**Pull request by linsang21**
**Issue #:** 

**Description of changes:**
Default to logger from logger provider when initiate bgBlur, bgReplacement processors

**Testing**
1. Have you successfully run `npm run build:release` locally? yes

2. How did you test these changes? Test using react meeting demo

3. If you made changes to the component library, have you provided corresponding documentation changes? 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/providers/BackgroundBlurProvider/index.tsx [hunk 1]
```diff
@@ -158,12 +158,10 @@ export const BackgroundBlurProvider: FC<React.PropsWithChildren<Props>> = ({
     );
     const currentProcessor = await initializeBackgroundBlur();
     try {
-      const logger =
-        options?.logger ||
-        new ConsoleLogger('BackgroundBlurProvider', LogLevel.INFO);
+      const optionsLogger = options?.logger || logger;
```
Comment (ltrung) [inline src/providers/BackgroundBlurProvider/index.tsx:161]: This is after the initialize right? We pass the logger to DefaultVideoTransformDevice

File: src/providers/BackgroundBlurProvider/index.tsx [hunk 2]
```diff
@@ -51,7 +49,8 @@ export const BackgroundBlurProvider: FC<React.PropsWithChildren<Props>> = ({
   options,
   children,
 }) => {
-  const logger = useLogger();
+  let logger = useLogger();
+  logger = options?.logger || logger;
```
Comment (ltrung) [inline src/providers/BackgroundBlurProvider/index.tsx:53]: Why not just one line `let logger = options?.logger || useLogger()` ?

**Review by xuesichao — APPROVED** [review]

File: src/providers/BackgroundBlurProvider/index.tsx (hunk 2 above)
Comment (linsang21) [reply src/providers/BackgroundBlurProvider/index.tsx:53]: there's warning for conditionally calling hooks

Comment (georgezy-amzn) [reply src/providers/BackgroundBlurProvider/index.tsx:53]: how about `let logger = options?.logger ?? useLogger()` ?

Comment (linsang21) [reply src/providers/BackgroundBlurProvider/index.tsx:53]: same warning

**Pull request merged**